p2020mockup/cr-6.2.8.2.html

```diff
@@ -45,16 +45,10 @@ <h3>Draft</h3>
             (Microsoft Corp.)
            (Until February 2014)</dd>
 
-        <dt>Participate:</dt><dd>
-      <a href="https://github.com/w3c/hr-time/">GitHub w3c/hr-time</a>
-    </dd><dd>
-      (@@required) <a href="https://github.com/w3c/hr-time/issues/">File a bug</a>
-    </dd><dd>
-      <a href="https://github.com/w3c/hr-time/commits/gh-pages">Commit history</a>
-    </dd><dd>
-      <a href="https://github.com/w3c/hr-time/pulls/">Pull requests</a>
-    </dd>
-      <dt class="caniuse-title">Browser support:</dt><dd class="caniuse-stats"><a href="https://caniuse.com/high-resolution-time">caniuse.com</a></dd></dl>
+           <dt>Feedback:</dt>
+           <dd><a href="https://github.com/w3c/hr-time/issues">https://github.com/w3c/hr-time/issues</a></dd>
+           <dd><a href="mailto:public-web-perf@w3.org?subject=%5Bhr-time%5D%20YOUR%20TOPIC%20HERE">public-web-perf@w3.org</a> with subject line “<kbd>[hr-time] <var>… message topic …</var></kbd>” (<a href="http://lists.w3.org/Archives/Public/public-web-perf/" rel="discussion">archives</a>)</dd>
+     </dl>
 
```

Comment:
Is the "feedback" `<dt>` meant to be illustrative or normative?

I'm not convinced that we should push in-your-face URIs for github issues

Reply:
I could go either way. I'm not sure what text I'd replace it with that would as conveniently convey that it's the issue-tracker of Github repo XYZ, and certainly it's consistent with the way we use URLs in the header generally. I wouldn't mind leaving the link text customizable, and having specberus only check the URL, though.

I *do* want to compact the entire header into something more user-friendly, that takes better advantage of this newfangled thing called hyperlinks (and CSS), but that's maybe a separate project. Maybe for 2021? :)